packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -99,6 +103,18 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
 
   public componentDidMount(): void {
     this._fitParentContainer();
+    if (this.props.showYAxisLables) {
```

Comment:
this prop is applicable only for horizontal bar chart right?

Reply:
yes, I have added a check for charttype as well.